TextAnalysis/src/lang/anal/word/Word.java

```diff
@@ -5,31 +5,46 @@
 public class Word {
 
 	private String value; //The word itself
-	private String definition;
+	private LinkedList<String> definitions;
 	private String POS; //Part of speech 'verb' , 'noun' , 'adjective', etc.
 	private LinkedList<String> synonyms,antonyms;
```

Comment:
Are the synonyms/antonyms the same for words with the same spelling but different definition? In other words, do the different definitions on line 8 each have their own synonyms/antonyms?